oxford-metadata.ttl

```diff
@@ -523,50 +527,51 @@
 # Detailed extra-cellular concentrations
 :extracellular_bath_potassium_concentration
     a :Annotation ;
-    a :ExtracellularPotassiumConcentration ;
-    rdfs:label "Potassium concentration in the extracellular bath, far from the cell" .
+    a :ExtracellularBathConcentration ;
```

Comment:
Question for @mirams here: do we also want a line
```
    a :extracellular_potassium_concentration ;
```
and include the `a rdfs:Class` for `:extracellular_potassium_concentration` etc? So that there's an explicit indication that this is a potassium concentration? In theory that would mean (when implemented) that clamping `extracellular_potassium_concentration` would also clamp the bath & cleft potassium concentrations if they existed in a model.

Reply:
I guess that'd also mean that requiring `extracellular_potassium_concentration` to exist would imply you wanted _at least one_ variable that matches?

Reply:
Ha! That's what Michael had to start with, if we can annotate so we are flexible and can do both "clamp extracellular bath concs" (getting bath sodium, bath potassium, bath calcium...) or "clamp extracellular potassium concs" (getting both the one pool or cleft/bath models) that would be nice.